install/kubernetes/helm/istio/values.yaml

```diff
@@ -446,3 +446,39 @@ kiali:
   dashboard:
     username: admin
     password: admin
+
+certmanager:
+  enabled: false
+  replicaCount: 1
+  image:
+    repository: quay.io/jetstack/cert-manager-controller
+    tag: v0.3.1
+    pullPolicy: IfNotPresent
+  clusterResourceNamespace: "istio-system"
```

Comment:
Not sure this is right - most others use a different style (check the other templates)